src/Features/Core/Portable/UnusedReferences/IUnusedReferencesService.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.UnusedReferences
+{
+    internal interface IUnusedReferencesService
+    {
+        /// <summary>
+        /// Determines unused references from this compilation.
+        /// </summary>
+        Task<ImmutableArray<UnusedReference>> GetUnusedReferencesAsync(
+            Project project,
+            Compilation compilation,
```

Comment:
Why does this need a compilation of it already takes a project?

Reply:
Removed